datalad_registry/static/main.css

```diff
@@ -64,7 +64,6 @@ div#datalad-registry div.pagination {
 
 .tooltip {
     position: relative;
-    display: inline-block;
```

Comment:
ping @jsheunis  as we torturing your baby here to fix #302 - may be there is a better solution?

Reply:
Can't give binary answer ;) -- I have not analyzed the cause of the bug, so can't attest that it happens only "where the file size column cells are empty" but "yes" -- after removing this added CSS it seems that it started to behave as desired.